pages.tr/common/docker-compose.md

```diff
@@ -0,0 +1,36 @@
+# docker-compose
+
+> Çoklu konteynerli docker uygulamalarını çalıştırın ve yönetin.
+> Daha fazla bilgi için: <https://docs.docker.com/compose/reference/overview/>.
+
+- Tüm konteynerleri listele:
+
+`docker-compose ps`
+
+- Mevcut dizinde bir `docker-compose.yml` dosyası çalıştırarak arkaplandaki tüm konteynerleri çalıştırın ve başlatın:
+
+`docker-compose up -d`
+
+- Tüm konteynerleri çalıştırın ve gerekiyorsa yeniden oluşturun:
+
+`docker-compose up --build`
+
+- Tüm konteynerleri alternatif bir beste dosyasıyla başlatın:
+
+`docker-compose --file {{path/to/file}} up`
```

Comment:
Will you please translate `path/to/file`?

Reply:
Sure